src/service/action-impl.js

```diff
@@ -323,12 +323,6 @@ export class ActionService {
               event,
               ActionTrust.HIGH
             );
-            // Only if the element has an action do we prevent the default.
-            // In the absence of an action, e.g. on="[event].method", we do not
-            // want to stop default behavior.
-            if (hasAction) {
-              event.preventDefault();
```

Comment:
Aha! Looks like the original `preventDefault` was added because [`SPACE` will scroll the page otherwise](https://github.com/ampproject/amphtml/pull/8879#discussion_r113768136). In that case, we should keep this line if the event comes from that key specifically.

Unfortunately this also has the consequence of preventing other types of events which may actually be desirable...